client/Wayland/wlfreerdp.c

```diff
@@ -315,20 +333,18 @@ static int wlfreerdp_run(freerdp* instance)
 		return -1;
 	}
 
-	handle_uwac_events(instance, context->display);
-
 	while (!freerdp_shall_disconnect(instance))
 	{
 		handles[0] = context->displayHandle;
-		count = freerdp_get_event_handles(instance->context, &handles[1], 63);
```

Comment:
I don't understand in which case freerdp_get_event_handles() returns no event handle.

Reply:
Not currently, correct. But this is client code and the core library may change over time, so better safe than sorry